package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "masq-common",
+  "version": "1.0.0",
+  "description": "Common functions for Masq",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "babel ./src/ --presets env --out-dir dist/ && webpack --mode development",
+    "test": "eslint test/*.js && npm run build && mocha-headless-chrome -f ./test/index.html -a no-sandbox -a disable-setuid-sandbox",
+    "start": "http-server ."
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-runtime": "^6.26.0",
+    "chai": "^4.1.2",
+    "eslint": "^5.7.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jest": "^22.1.2",
+    "eslint-plugin-mocha": "^5.2.0",
+    "eslint-plugin-node": "^8.0.0",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-standard": "^4.0.0",
+    "mocha": "^5.2.0",
+    "mocha-headless-chrome": "^2.0.1",
+    "uglify-js": "^3.1.10",
+    "webpack": "^4.20.2",
+    "webpack-cli": "^3.1.2"
+  },
+  "dependencies": {
+    "bluebird": "^3.5.3",
+    "http-server": "^0.11.1",
+    "hyperdb": "^3.5.0",
+    "random-access-idb": "^1.2.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/QwantResearch/masq-common.git"
+  },
+  "keywords": [
+    "encryption",
+    "common",
+    "utils"
+  ],
+  "author": "Levent DEMIR",
+  "license": "ISC",
```

Comment:
We use MIT in masq-app, at least for now, IDK if we should use MIT here too ?

Reply:
ok done